great-idea-website/css/index.css

```diff
@@ -130,13 +156,30 @@ header img {
 	font-size: 16px;
 }
 
+@keyframes button-animation{
+	0% {width: 50%;}
+	100% {width: 100%;}
+}
+
+.main-header button:hover {
+	animation-name: button-animation;
+	animation-duration: 1s;
+	animation-fill-mode: forwards;
```

Comment:
Very clever and good use combing this with the other aspects of the page animations and hover states.